tests/features/bootstrap/FeatureContext.php

```diff
@@ -6,13 +6,19 @@
  */
 
 use Behat\Behat\Context\SnippetAcceptingContext;
+use Drupal\Core\Url;
 use Drupal\DrupalExtension\Context\RawDrupalContext;
+use Drupal\joinup\Traits\EntityTrait;
+use Drupal\joinup\Traits\UtilityTrait;
 
 /**
  * Defines generic step definitions.
  */
 class FeatureContext extends RawDrupalContext implements SnippetAcceptingContext {
 
+  use EntityTrait;
+  use UtilityTrait;
```

Comment:
Yes we do, some of our custom modules implement contexts where we (or we will) reuse these traits. I discussed with @pfrenssen during the implementation of this ticket and we agreed that it was a good approach to move functions that are not Behat steps to traits.